src/newcharacter.cpp

```diff
@@ -883,7 +883,7 @@ tab_direction set_stats(WINDOW *w, player *u, points_left &points)
             if (u->dex_max >= HIGH_STAT) {
                 mvwprintz(w, 3, iSecondColumn, c_ltred, _("Increasing Dex further costs 2 points."));
             }
-            mvwprintz(w_description, 0, 0, COL_STAT_BONUS, _("Melee to-hit bonus: +%d"),
+            mvwprintz(w_description, 0, 0, COL_STAT_BONUS, _("Melee to-hit bonus: +%f"),
```

Comment:
```+%.2f``` will be much better here. Currently, it looks quite ugly:
![capture](https://cloud.githubusercontent.com/assets/5421954/19622805/f0c95c66-98bb-11e6-85e9-9e03bca9a7b9.PNG)